tests/test_tasks.py

```diff
@@ -197,9 +207,9 @@ def test_remove_task(self):
         }
 
         registry = TaskRegistry()
-        registry.add('mytask', 'mock_backend', args)
-        registry.add('newtask', 'to_remove', None)
-        registry.add('atask', 'test_backend', None)
+        registry.add('mytask', 'mock_backend', 'acme', args)
```

Comment:
Better you change `acme` but another real `category`. Someone can think that parameter is not for setting the category of items to retrieve.